src/main/java/DukeException.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Represents errors that occurs as a result of running the program.
+ */
+public class DukeException extends Exception {
+    /**
+     * Instantiates a DukeException object.
+     *
+     * @param msg The specific error message.
+     */
+    public DukeException(String msg) {
```

Comment:
the naming of the variable "msg" is probably questionable

Reply:
Perhaps spelling "errorMessage" out in full may be better